### PR TITLE
fix: adjust expanded offer

### PR DESCRIPTION
**Pull request by 0x6762**
task: local-913
obs: test responsive view

### CODE REVIEW

File: app/src/utils/formatters.ts [hunk 1]
```diff
@@ -1,3 +1,15 @@
+export function formatTimeLimit(time: Date): string {
```
Comment (samb-local) [inline app/src/utils/formatters.ts:1]: We should add treatment when showing 1 hour, otherwise it'll say, 1 hours 30 minutes, instead of 1 hour 30 minutes.

**Review by samb-local — DISMISSED** [review]
Looks very good! Added a minor comment.

**Comment by samb-local** [comment]
Tested on 3 devices, 2 android, 1 iPhone, looked good on all of them.

File: app/src/utils/formatters.ts (hunk 1 above)
Comment (davirds) [reply app/src/utils/formatters.ts:1]: good catch! The same can be applied to the minutes, if we want to be perfectionists.

**Review by davirds — APPROVED** [review]
LGTM!

**Pull request merged**